spec/models/user/email_spec.rb

```diff
@@ -42,5 +42,16 @@
         expect(subject.errors[:email].count).to eq(1)
       end
     end
+
+    context 'when the email is not confirmed' do
+      subject { create(:user_email, :unconfirmed) }
+
+      it 'sends email with ActiveJob queue' do
+        message_delivery = instance_double(ActionMailer::MessageDelivery)
```

Comment:
@fonglh Can you check if you need this, will `expect(ActionMailer::MessageDelivery).to receive(:deliver_later)` work ?